### ISSUE TITLE
latest version of auctex require make install instead of simple file copy

### ISSUE THREAD

**Issue by LukeXuan** [doomemacs/doomemacs#7937]
### I confirm that...

- [X] I have searched the [issue tracker](https://github.com/doomemacs/doomemacs/issues), [documentation](https://docs.doomemacs.org), [FAQ](https://docs.doomemacs.org/-/faq), [Discourse](https://discourse.doomemacs.org), and [Google](https://google.com), in case this issue has already been reported/resolved.

- [X] I have read "[How to Debug Issues](https://doomemacs.org/d/how2debug)", and will use it to provide as much information about this issue as possible.

- [X] The issue can be reproduced on the **latest** available commit of Doom.

- [X] The issue can be reproduced on a stable release of Emacs, such as 27, 28, or 29. *(Unstable versions end in .50, .60, or .9x)*

### Expected behavior

With `latex` module enabled, `auctex` is properly installed.

### Current behavior

`tex-site.el` not found.

### Steps to reproduce

Just enable `latex`.
Also see here: https://github.com/radian-software/straight.el/issues/1165

### System Information

N/A

**Comment by LukeXuan** [comment]
My temporary fix (adapted from the upstream issue comments):
```
;; temporary fix for auctex
(package! auctex
  :recipe (:files ("*" (:exclude ".git"))
	   :pre-build ("make" "tex-site.el")))

```

**Comment by hlissner** [comment]
Already fixed in 96de02c76900768ae144019ff8d69737857ebac0.
